assignment.js

```diff
@@ -16,7 +16,11 @@ import {
    result === (4 + 8) - 3 * 6; // true
 */
 
-// ...
+function first (arg1, arg2) {
+  return function (argF1, argF2) {
+    return ((argF1 + argF2) - (arg1 * arg2));
```

Comment:
```suggestion
    return argF1 + argF2 - arg1 * arg2;
```

* return -- это ключевое слово, за которым следует выражение => внешние скобки не нужны
* оператор * имеет приоритет 14
* операторы + и - имеют приоритет 13

поэтому, если убрать скобки, выполнится сначала умножение, потом сложение и вычитание, т. к. у них одинаковый приоритет и они выполняются слева направо.

поэтому скобки можно убрать вовсе